### ISSUE TITLE
There is no column with name 'sync_in_background' on table 'oc_mail_mailboxes'.

### ISSUE THREAD

**Issue by hardwareadictos** [nextcloud/mail#6422]
### Steps to reproduce

1. Installing Mail App

### Expected behavior

App installing correctly

### Actual behavior

App doesn't get installed reporting missing columns on the DB

### Mail app version

1.12.0	

### Operating system

Docker on Debian 11

### PHP engine version

PHP 8.0

### Web server

Nginx

### Database

MySQL

### Additional info

I don't know from which version is failing because it's the first time i try to install it, not sure if it's really an app problem or a DB problem

![image](https://user-images.githubusercontent.com/20403906/168047917-3d853981-b031-4c33-af7c-fe5de9434573.png)

**Comment by miaulalala** [comment]
Do you have access to your nextcloud.log? If so, can you please check for any log entries that point towards Mail and Migrations?

**Comment by hardwareadictos** [comment]
> 

Just forced the error but nothing seems to appear on logs:

![imagen](https://user-images.githubusercontent.com/20403906/170261406-16f53924-b5da-40cc-a494-ca9e93734780.png)

**Comment by hardwareadictos** [comment]
Is there anything more i can provide to help?

**Comment by miaulalala** [comment]
> Is there anything more i can provide to help?

yes, your debug log files. can you turn on [debug mode](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html) for your instance and reinstall mail?

**Comment by hardwareadictos** [comment]
Enable debug mode on config:

'loglevel' => 0,

nothing seems to appear on logs:

[log.txt](https://github.com/nextcloud/mail/files/11053413/log.txt)

**Comment by hardwareadictos** [comment]
Nothing? This issue has more than a year.

**Comment by miaulalala** [comment]
Can you check your oc_migrations table with 

`select * from oc_migrations where app = 'mail;` and post the output?

**Comment by hardwareadictos** [comment]
> Can you check your oc_migrations table with
> 
> `select * from oc_migrations where app = 'mail';` and post the output?

Sure! Will post here the results later.

**Comment by hardwareadictos** [comment]
Here you have it:

```
MariaDB [nextcloud]> select * from oc_migrations where app = 'mail';
+------+------------------------+
| app  | version                |
+------+------------------------+
| mail | 0100Date20180825194217 |
| mail | 0110Date20180825195812 |
| mail | 0110Date20180825201241 |
| mail | 0130Date20190408134101 |
| mail | 0156Date20190828140357 |
| mail | 0161Date20190902103559 |
| mail | 0161Date20190902103701 |
| mail | 0161Date20190902114635 |
| mail | 0180Date20190927124207 |
| mail | 0190Date20191118160843 |
| mail | 0210Date20191212144925 |
| mail | 1020Date20191002091034 |
| mail | 1020Date20191002091035 |
| mail | 1020Date20200206134751 |
| mail | 1030Date20200228105714 |
| mail | 1040Date20200422130220 |
| mail | 1040Date20200422142920 |
| mail | 1040Date20200506111214 |
| mail | 1040Date20200515080614 |
| mail | 1040Date20200529124657 |
| mail | 1050Date20200624101359 |
| mail | 1050Date20200831124954 |
| mail | 1050Date20200921141700 |
| mail | 1050Date20200923180030 |
| mail | 1060Date20201015084952 |
| mail | 1080Date20201119084820 |
| mail | 1080Date20210108093802 |
| mail | 1090Date20210127160127 |
| mail | 1090Date20210216154409 |
+------+------------------------+
29 rows in set (0.005 sec)
```

**Comment by miaulalala** [comment]
And you're trying to install / have installed which version of mail at the moment?

**Comment by hardwareadictos** [comment]
This is the result:

````
root@CloudTest-DBN01:/mnt/docker-lib/docker-compose/nextcloud# docker exec -u www-data Nextcloud-TEST php occ app:update mail
mail is up-to-date or no updates could be found
````

I'm trying to install the last one available on the Application store on Nextcloud, issue is being there for the last 2 years so i cannot remember which was the first update that gave isssues.

**Comment by hardwareadictos** [comment]
> did you install the app via app store?

Yes, that's what i did. I always install apps via App Store inside Nextcloud.

**Comment by miaulalala** [comment]
OK, can you uninstall the mail app completely, set your log to debug and then re- install the mail app and keep an eye on the logs? Let's see what they say.

**Comment by hardwareadictos** [comment]
Unistall via occ went fine. Install reports the same:

```
Ha ocurrido un error durante la solicitud. No se puede continuar.
Database error when running migration 1096Date20210407150016 for app mail There is no column with name "sync_in_background" on table "oc_mail_mailboxes".
````

That's what front end reports, i enabled logging atl level 0 (DEBUG) but i don't see nothing on logs. I proceeded with a restart between remove and install.

**Comment by nextcloud-command** [comment]
Hi there! It looks like this issue hasn’t seen much activity for a while and might need a bit more information to move forward. If we don’t hear back soon, we may close it to keep things organized. Thanks again for your contributions – we really appreciate it!

**Comment by kesselb** [comment]
Hi, 

I assume we cannot replicate how your installation ended up in that state. One database table is missing a column, but there's a migration for the column, so that might be an interrupted update or so. 

My suggestion would be to uninstall the app, reset the app (https://github.com/nextcloud/mail/blob/main/doc/developer.md#resetting-the-app) and install again. Note: This will remove all existing data for the mail app.

**Comment by nextcloud-command** [comment]
Hi there! It looks like this issue hasn’t seen much activity for a while and might need a bit more information to move forward. If we don’t hear back soon, we may close it to keep things organized. Thanks again for your contributions – we really appreciate it!
